ckb-bench/src/bench.rs

```diff
@@ -225,23 +274,138 @@ impl TransactionProducer {
                     // SendError occurs, the corresponding transaction receiver is dead
                     return;
                 }
+                count += 1;
+                duration_count += 1;
+                if Instant::now().duration_since(start_time) >= Duration::from_secs(log_time) {
+                    ckb_testkit::info!("[TransactionProducer] producer count: {} liveCell producer remaining :{} ,duration time:{}s, duration tps:{} ", count,live_cell_receiver.len(),log_time,duration_count/log_time);
```

Comment:
1. `log_time: u64` doesn't intuitively represent its purpose, how about  `log_duration: Duration`?

3. And I think `duration tpc` should be `count / start_time.elapsed()`, and `duration time:` should be `start_time.elapsed()`